tests/qgis-projects/tests/test_print.qgs.cfg

```diff
@@ -1,20 +1,22 @@
 {
     "metadata": {
-        "qgis_desktop_version": 31611,
-        "lizmap_plugin_version": "master",
-        "lizmap_web_client_target_version": 30400,
+        "qgis_desktop_version": 31616,
+        "lizmap_plugin_version_str": "3.8.3",
```

Comment:
Even Lizmap developers are not upgrading their QGIS desktop lizmap plugin :)

Reply:
I don't use it often enough anymore